TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -133,6 +133,8 @@ import UIKit
             return
         }
 
+        precondition(!TradeItDeviceManager.isDeviceJailBroken(), "ERROR: the device is jailbroken, couldn't load the TradeItSDK !")
```

Comment:
This will just totally crash the app. Do we have to crash? Can we just deny access to any TradeIt functionality? Partners don't want us to ever be the cause of their app crashing.

Reply:
@guillaumedebavelaere can we just put this in front of the linking so that anything trying to open the oAuth url will just issue an error message and never actually request a URL from our server?